sale_order_type/models/account_move.py

```diff
@@ -21,9 +21,12 @@ class AccountMove(models.Model):
     @api.depends("partner_id", "company_id")
     def _compute_sale_type_id(self):
```

Comment:
@igallart In fact, you should add an  @api.depends_context() decorator with all keys in the function.

You should add also a key 'company'.

Reply:
@rousseldenis , thank you very much for your comment, but what I have done is to correct a particular error that was currently failing, my intention is not to refactor the module, I have adapted the correction to what was there, I think that being a version 12 is not necessary to invest so much time, if it were for a newer version or in the process of migration if it should be done. If someone wants to make the improvements will be welcome.

Reply:
Sorry, is version 14 not 12, but my objective of this pull request is to correct a specific error, not to refactor. The original module also does not make use of api_depends_context...

Reply:
@rousseldenis , add api_depends_context.